test/integration/organization_test.rb

```diff
@@ -3,6 +3,8 @@
 class OrganizationIntegrationTest < ActionDispatch::IntegrationTest
   def setup
     FactoryGirl.create(:host)
+    Capybara.current_driver = Capybara.javascript_driver
```

Comment:
If you only need JS for this test, do this in the specific test - otherwise it'll slow down other tests in here (Poltergeist is slower than rack-test) and you don't need to login twice